packages/blade/src/tokens/global/fontFamily/fontFamily.web.ts

```diff
@@ -2,6 +2,8 @@ import type { FontFamily } from './types';
 
 export const fontFamily: FontFamily = {
   text:
-    'Lato, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"',
-  code: 'monospace',
+    '"Inter", -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif',
```

Comment:
the fallback remains as old one apple-system?

Reply:
yes. Its similar.